cmd/thanos/store.go

```diff
@@ -49,7 +50,15 @@ func registerStore(m map[string]setupFunc, app *kingpin.Application, name string
 	blockSyncConcurrency := cmd.Flag("block-sync-concurrency", "Number of goroutines to use when syncing blocks from object storage.").
 		Default("20").Int()
 
+	selectorLabels := cmd.Flag("store.selector-label", "Store Gateway selector labels that will be exposed in info endpoint (repeated).").
```

Comment:
hm we have 
```
	selectorLabels := cmd.Flag("selector-label", "Query selector labels that will be exposed in info endpoint (repeated).").
		PlaceHolder("<name>=\"<value>\"").Strings()
```

Maybe worth to be consistent with naming and remove `store` prefix?  essentially it has not much to do with store API for example.